.github/workflows/devcontainer.yml

```
@@ -0,0 +1,56 @@
+name: Build for Development
```

```suggestion
name: devcontainer
```
Maybe this is more descriptive?